### PR TITLE
2022-08-01 AMM-EpiInvert

### PR DESCRIPTION
**Pull request by lalvarezmat**
This is our first contribution. For additional information about our model see: 

https://github.com/lalvarezmat/EpiInvert

Thanks 
Luis Alvarez

### CODE REVIEW

**Review by Bisaloo — CHANGES_REQUESTED** [review]
Hi @lalvarezmat, thanks for your contribution! We are always happy to welcome new contributors to the European Covid-19 Forecast Hub.

We have made small changes to the metadata files recently:

- I have made a commit suggestion to add more structure to the `model_contributors` field
- Could you move the metadata file to the [`model-metadata/` folder at the root of the GitHub repository](https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe/tree/main/model-metadata)?
- Could you remove the `metadata-` prefix in the name of your metadata file?
- Could you use the `.yml` extension instead of the `.txt` extension?

We merge all submissions together on Tuesdays, and the visualisations on the [website](https://covid19forecasthub.eu/index.html) are updated shortly after that. After four weeks of submissions, you will also see your model in the [evaluation reports](https://covid19forecasthub.eu/reports.html).

You are very welcome to join our mailing list, which we use to organise meetings for all participants in the hub. If you are interested, please request to join here: [groups.google.com/u/1/g/euro-covid19-forecast-hub](https://groups.google.com/u/1/g/euro-covid19-forecast-hub). You can also [join our slack workspace](https://join.slack.com/t/europeanmodellinghub/shared_invite/zt-1czriyffn-ZZDVoL14wndReZftQ6tJQw).

**Comment by lalvarezmat** [comment]
Dear Hugo, 

Thank for your help. I followed your suggestions and made commits. Do I have to repeat the "Pull Request"? 

By the way, looking at the visualization of the forecast I uploaded it seems that it does not corresponds to the data I uploaded. For instance for France I uploaded for the next 4 weeks : 275112, 231675, 222159 and 246320 but the data visualization can not corresponds to these data. 

Regards
Luis

File: model-metadata/AMM-EpiInvert.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+website_url: https://github.com/lalvarezmat/EpiInvert
+license: cc-by-4.0
+team_model_designation: primary
+methods: Learning from the past a short time forecast of the COVID-19 incidence curve trend.
+methods_long: We design a learning procedure, EpiInvertForecast,  to forecast the future trend of the daily incidence. 
+This learning procedure is based on a database of 27,418 incidence trend curves computed in the past, with EpiInvert, 
+using real data. The forecast of the current incidence trend is obtained as a weighted average of the 27,418 database 
+curves where the weight of each database curve depends on the similarity between the current curve and the database 
+curve in the past. We also compute empirical confidence intervals for the forecast estimation and, using the weekly
+seasonality computed by EpiInvert, we also estimate a forecast of the original incidence curve.
```
Comment (Bisaloo) [inline model-metadata/AMM-EpiInvert.yml:23]:
```suggestion
methods_long: We design a learning procedure, EpiInvertForecast,  to forecast the future trend of the daily incidence. 
  This learning procedure is based on a database of 27,418 incidence trend curves computed in the past, with EpiInvert, 
  using real data. The forecast of the current incidence trend is obtained as a weighted average of the 27,418 database 
  curves where the weight of each database curve depends on the similarity between the current curve and the database 
  curve in the past. We also compute empirical confidence intervals for the forecast estimation and, using the weekly
  seasonality computed by EpiInvert, we also estimate a forecast of the original incidence curve. 
```

File: model-metadata/AMM-EpiInvert.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+team_model_designation: primary
+methods: Learning from the past a short time forecast of the COVID-19 incidence curve trend.
+methods_long: We design a learning procedure, EpiInvertForecast,  to forecast the future trend of the daily incidence. 
+This learning procedure is based on a database of 27,418 incidence trend curves computed in the past, with EpiInvert, 
+using real data. The forecast of the current incidence trend is obtained as a weighted average of the 27,418 database 
+curves where the weight of each database curve depends on the similarity between the current curve and the database 
+curve in the past. We also compute empirical confidence intervals for the forecast estimation and, using the weekly
+seasonality computed by EpiInvert, we also estimate a forecast of the original incidence curve. 
+citation: https://doi.org/10.1073/pnas.2105112118 , https://doi.org/10.3390/biology11040540 , 
+https://ctim.ulpgc.es/covid19/EpiInvertForecast.html
```
Comment (Bisaloo) [inline model-metadata/AMM-EpiInvert.yml:25]:
```suggestion
  https://ctim.ulpgc.es/covid19/EpiInvertForecast.html
```

**Review by Bisaloo — CHANGES_REQUESTED** [review]
Thanks. I have two other suggestions related to the yaml syntax and the checks should now pass.

For your point regarding the preview app, it's a long standing issue where point forecasts were not displayed (https://github.com/covid19-forecast-hub-europe/HubSubmissionApp/issues/12). Your comment gave me the necessary push to finally fix this issue. Your forecasts should now display fine.

**Comment by lalvarezmat** [comment]
Dear Hugo, 

Thanks again for your help, now the visualization looks fine, I accepted and commit your modification proposal but some submission validation errors remain. 

Regards
Luis

**Pull request merged**